skunkworks_crow/src/main/java/org/odk/share/activities/AboutActivity.java

```diff
@@ -37,6 +38,10 @@ protected void onCreate(Bundle savedInstanceState) {
         setTitle(getString(R.string.about));
         setSupportActionBar(toolbar);
 
+        setSupportActionBar(toolbar);
+        ActionBar ab = getSupportActionBar();
```

Comment:
can directly be written as `getSupportActionBar().setDisplayHomeAsUpEnabled(true);`